api/sql/schema/750-rule.decision.lookup.sql

```diff
@@ -88,7 +88,7 @@ BEGIN
     FROM
         [integration].[vAccount]
     WHERE
-        accountNumber = @destinationAccount AND
+        (accountNumber = @destinationAccount OR @destinationAccount IS NULL) AND
```

Comment:
Allow calling with account owner only, as there may be no accounts at all and rule still can be used